#! /usr/bin/env python
PACKAGE='imu_ekf_example' # Name of package here

from dynamic_reconfigure.msg import SensorLevels
from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

#       Name          Type      Level  Description                Default  Min    Max
gen.add("q_quat",     double_t, 0,    "Process noise covariance", 10.0,    0.0,   10.0)
gen.add("q_rate",     double_t, 0,    "Process noise covariance", 10.0,    0.0,   10.0)
gen.add("r_accel",    double_t, 0,    "Measurement covariance",   0.01,    0.0,   10.0)
gen.add("r_gyro",     double_t, 0,    "Measurement covariance",   0.01,    0.0,   10.0)

# Put name of cfg file in the last argument of 'exit'
exit(gen.generate(PACKAGE, PACKAGE, "ImuEkfExample"))
